### ISSUE TITLE
Why is credentails.to_h/to_json empty when there are credentials?

### ISSUE THREAD

**Issue by dorianmariefr** [rails/rails#48554]
```ruby
>> Rails.application.credentials.to_h
=> {}
>> Rails.application.credentials.to_json
=> "{}"
```

ruby 3.2.2 (2023-03-30 revision e51014f9c0) +YJIT [arm64-darwin22]

Rails 7.1.0.alpha

**Comment by skipkayhil** [comment]
Hey @dorianmariefr, thanks for opening an issue! I proposed a fix in #48556, feel free to test it out and let me know if it works

**Comment by dorianmariefr** [comment]
❤️
